cmd/node/main.go

```diff
@@ -669,15 +676,21 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 		networkComponents,
 		uint64(ctx.GlobalUint(bootstrapRoundIndex.Name)),
 		version,
+		elasticIndexer,
 	)
 	if err != nil {
 		return err
 	}
 
+    if shardCoordinator.SelfId() == sharding.MetachainShardId {
+        indexValidatorsListIfNeeded(elasticIndexer, nodesCoordinator)
+    }
+
 	vmAccountsDB, err := hooks.NewVMAccountsDB(
 		stateComponents.AccountsAdapter,
 		stateComponents.AddressConverter,
 	)
+
```

Comment:
remove empty line. leave it as it was before

Reply:
Done.